integration_tests/base_routes.py

```diff
@@ -41,239 +39,412 @@ def message():
     return "Hello world, from ws"
 
 
-callCount = 0
+# ===== Lifecucle handlers =====
+
+
+async def startup_handler():
+    print("Starting up")
+
+
+@app.shutdown_handler
+def shutdown_handler():
+    print("Shutting down")
```

Comment:
are we using these anymore with the new tests?

Reply:
It is still registered for the app, but I couldn't find a way to test they are working correctly. I would like them to imply a side effect I could then check in the test. I left it here for now until someone finds a way to actually write a test for it.